src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsinterventionsservice/controller/ReferralController.kt

```diff
@@ -121,9 +122,16 @@ class ReferralController(
   @GetMapping("/sent-referrals/summary/service-provider")
   fun getServiceProviderSentReferralsSummary(
     authentication: JwtAuthenticationToken,
+    @Nullable @RequestParam(name = "dashboardType", required = false) dashboardTypeSelection: String?,
   ): List<ServiceProviderSentReferralSummaryDTO> {
     val user = userMapper.fromToken(authentication)
-    return referralService.getServiceProviderSummaries(user).map { ServiceProviderSentReferralSummaryDTO.from(it) }
+    var dashboardType: DashboardType?
```

Comment:
nit: you can assign the try catch expression to avoid using mutable variables.

i.e. val dashboardType = try { ...

Reply:
done